libs/interpret-text/src/lib/TextExplanationDashboard/Control/TextExplanationView/ITextExplanationViewSpec.ts

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { IChoiceGroupOption, IStackTokens } from "@fluentui/react";
+import { localization } from "@responsible-ai/localization";
+
+import { QAExplanationType, RadioKeys } from "../../CommonUtils";
+
+export interface ITextExplanationViewState {
+  /*
+   * Holds the state of the dashboard
+   */
+  maxK: number;
+  topK: number;
+  radio: string;
+  // qaRadio?: string;
```

Comment:
remove commented qaRadio?

Reply:
I will add the functionality in the upcoming PR, so left it as commented code.